src/pages/home/report/ReportActionsList.tsx

```diff
@@ -709,13 +712,19 @@ function ReportActionsList({
     // When performing comment linking, initially 25 items are added to the list. Subsequent fetches add 15 items from the cache or 50 items from the server.
     // This is to ensure that the user is able to see the 'scroll to newer comments' button when they do comment linking and have not reached the end of the list yet.
     const canScrollToNewerComments = !isLoadingInitialReportActions && !hasNewestReportAction && sortedReportActions.length > 25 && !isLastPendingActionIsDelete;
+    const [reportActionsListTestID, reportActionsListFSClass] = getChatFSAttributes(participantsContext, 'ReportActionsList', report);
+
```

Comment:
`ReportActionsList` is being rerender many times, when switching and when sending/receiving  messages, I would recommend to wrap it in a `useMemo` and the dependency array would be `[report.reportID]`
<img width="292" alt="Screenshot 2024-11-13 at 11 44 20" src="https://github.com/user-attachments/assets/0da8b010-aa07-444c-8084-c716492e1c9f">

<img width="304" alt="Screenshot 2024-11-13 at 11 45 56" src="https://github.com/user-attachments/assets/92e7e769-791f-4a12-8ea8-d6099dddc454">